content/en/events/2023/kcsna/faq.md

```diff
@@ -0,0 +1,119 @@
+
+[Location & Venue]: /events/2023/kcsna/location/
+
+### Will I be able to attend remotely?
+
+No, based on our experience with previous Contributor Summits, we do not expect to be able
+to support remote access for this event.
+
+### What are the Health and Safety protocols?
+
+As a co-located event, the summit is required to adhere to KubeCon’s
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/health-and-safety/" rel="noopener noreferrer" target="_blank">
+Health & Safety Guidelines</a>. There may be additional restrictions that
+develop closer to the event, and we will update the event accordingly to follow
+venue, municipality and CDC guidelines at a minimum.
+
+### Why do I need to be a Kubernetes Org member to attend in-person?
+
+We welcome everyone to start contributing to the project, but with 1600+
+Kubernetes Org members and limited space and budget requirements we've
+made the decision to limit attendance to those that are already actively
+engaged with the project.
+
+If you are not yet a member, but active within the project, please consider
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information. If your application to join is pending, please
+contact us at summit-team@kubernetes.io about registering.
+
+Kubernetes Orgs:
+
+<ul>
+<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
+<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
+<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
+<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
+</ul>
+
+
+[org membership]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
+
+### What is a sponsored attendee?
+
+If you are a SIG lead or subproject owner and there is someone that is not yet
+an org member or from another project that might take part in a panel,
+SIG face-to-face meeting etc, you can sponsor them to attend the summit.
+Please [email us] to make arrangements.
+
+### I have an idea for a session
+
+Most of the Summit will be unconference-style, so you will have an opportunity
+to propose a session closer to the Summit itself. If you have an idea for a talk,
+workshop or panel please post your idea to the unconference issue or the CFP
+form once they become available.  
+
+### My SIG wants to meet at the Summit
+
+We do not know yet if we will be able to request additional space outside of
+the colo event space on Tuesday, November 7th. We will update this page and send
+out community wide messages about it once the details are available.
+
+### Will there be a doc sprint?
+
+Yes, SIG-Docs plans to hold a Doc Sprint at the Summit, which will run for the
+full day from 11am to 5pm.
```

Comment:
```suggestion
full day from 11am to 4pm.
```
I'm not sure what time the awards ceremony starts but in the past it has been around 4pm